mcp_clickhouse/mcp_env.py

```diff
@@ -120,22 +115,23 @@ def get_client_config(self) -> dict:
 
         return config
 
-    def _validate_required_vars(self) -> None:
-        """Validate that all required environment variables are set.
+    def _set_default_vars(self) -> None:
+        """Set default values for environment variables if they are not already set."""
+        defaults = {
+            "CLICKHOUSE_HOST": "localhost",
+            "CLICKHOUSE_USER": "",
+            "CLICKHOUSE_PASSWORD": "",
+            "CLICKHOUSE_PORT": "8123",
+            "CLICKHOUSE_SECURE": "false",
+            "CLICKHOUSE_VERIFY": "false",
+            "CLICKHOUSE_CONNECT_TIMEOUT": "5",
+            "CLICKHOUSE_SEND_RECEIVE_TIMEOUT": "300",
+        }
 
-        Raises:
-            ValueError: If any required environment variable is missing.
-        """
-        missing_vars = []
-        for var in ["CLICKHOUSE_HOST", "CLICKHOUSE_USER", "CLICKHOUSE_PASSWORD"]:
+        for var, default_value in defaults.items():
             if var not in os.environ:
-                missing_vars.append(var)
-
-        if missing_vars:
-            raise ValueError(
-                f"Missing required environment variables: {', '.join(missing_vars)}"
-            )
+                os.environ[var] = default_value
```

Comment:
This ends up calling `putenv` underneath. Ideally we wouldn't mutate process environment variables. Especially since this changes getenv to always be used with a default, it seems unnecessary